xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -409,9 +565,16 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Read the next JSON token from the input source.</summary>
+        <returns><see langword="true" /> if the token was read successfully; otherwise, <see langword="false" />.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.Text.Json.JsonReaderException">
+            An invalid JSON token is encountered.
```

Comment:
Is the indentation here fine even though the `-or-` has no indentation?

Reply:
Yes. It works fine like this.